rpcsvc/Makefile.am

```diff
@@ -12,5 +12,5 @@ nodist_rpcsvc_HEADERS = klm_prot.h nlm_prot.h rstat.h spray.h \
 	nfs_prot.h rquota.h sm_inter.h
 
 %.h: %.x
-	$(top_builddir)/rpcgen/rpcgen -h -o $@ $<
+	CPP='$(CC) -E -x c-header' $(top_builddir)/rpcgen/rpcgen -Y $(top_srcdir)/rpcgen -h -o $@ $<
```

Comment:
Actually I can't tell as I've just used what glibc does and thought it was the correct thing since rpcsvc-proto was slip out from glibc and it worked to solve the problem we're running into. The relevant code with a comment description is here:
https://sourceware.org/git/?p=glibc.git;a=blob;f=sunrpc/Makefile;h=f0bfd062b0312a44eca5623b15779cfe9fde269c;hb=refs/heads/release/2.29/master#l185

But instead of 
`CPP='$(CC) -E -x c-header' $(top_build...`
just
`CPP=${CPP} $(top_build...`
also seems to do the trick for me. I can update the pull request accordingly if you think that is the correct approach.

Reply:
Just `$(top_builddir)/rpcgen/rpcgen -Y $(top_srcdir)/rpcgen -h -o $@ $<` also seems to be enough for it to work.